docs/components/board/upboard.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Configure an upboard board"
+linkTitle: "upboard"
+weight: 70
+type: "docs"
+description: "Configure an upboard board."
+images: ["/icons/components/board.svg"]
+tags: ["board", "components"]
+# SMEs: Susmita
+---
+
+Configure a `upboard` board to integrate an Intel-based board like the [UP4000](https://github.com/up-board/up-community/wiki/Pinout_UP4000) into your robot:
+
+{{< tabs name="Configure an upboard Board" >}}
+{{% tab name="Config Builder" %}}
+
+Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+Click on the **Components** subtab and navigate to the **Create component** menu.
+Enter a name for your board, and select the type `board`.
+Next, select the **Model** card.
+Type in `upboard` and hit enter to select the `upboard` model.
```

Comment:
this is necessary bc it's not pre-populated right now but is still built into the rdk

Reply:
I just added the the model to the register. If you can check if the model shows up on the latest branch, that would be great!